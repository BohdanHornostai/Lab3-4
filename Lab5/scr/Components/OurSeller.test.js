import React from 'react'
import {render, cleanup} from '@testing-library/react'
import OurSeller from './OurSeller'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<OurSeller />)

    expect(asFragment(<OurSeller />)).toMatchSnapshot()
   })