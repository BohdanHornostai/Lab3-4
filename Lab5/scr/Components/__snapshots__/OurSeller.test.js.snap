// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <figure
      class="fir-image-figure"
    >
      <a
        class="fir-imageover"
        href="/userProfile"
        rel="noopener"
        target="_blank"
      >
        <img
          alt="David East - Author"
          class="fir-author-image fir-clickcircle"
          src="https://picsum.photos/200/150/?random"
        />
        <div
          class="fir-imageover-color"
        />
        <img
          class="fir-imageover-image fir-clickcircle"
          src="https://picsum.photos/200/150/?random"
        />
      </a>
      <figcaption>
        <div
          class="fig-author-figure-title white-text red"
        >
          Author
        </div>
        <div
          class="small grey"
        >
          Last seen 10 min ago
        </div>
      </figcaption>
    </figure>
  </div>
</DocumentFragment>
`;
